### PR TITLE
DOCS-5806 - Docs trace queries private beta

### PR DESCRIPTION
**Pull request by dussault-antoine**
### What does this PR do?

Start docs for trace queries
Task tracked [here](https://datadoghq.atlassian.net/browse/DOCS-5806)
Superseding https://github.com/DataDog/documentation/pull/18832/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/trace_queries/_index.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+The Flow Map helps you understand the request path and service dependencies from the resulting traces that match the Trace Query. Use the map to identify error paths, unusual service dependencies, or abnormally high request rates to a database.
+
+**Note**: The Flow Map is powered by [a sample of the ingested traffic](#the-data-that-trace-queries-are-based-on).
+
+Service nodes that match span queries are highlighted to show you which parts of the trace your query conditions are targeting.
+
+To get more information about **a single service**, hover on the service's node to see its metrics for request rate and error rate. To see metrics for the request rate and the error rate **between two services**, hover on an edge connecting the two services.
+
+Click with the map nodes to filter our traces that do not contain a dependency on the targeted service.
```
Comment (kayayarai) [inline content/en/tracing/trace_queries/_index.md:68]: I couldn't quite figure out what this was saying. Is it something like "To filter out traces that do not contain a dependency (or "aren't involved" on the targeted service (do something)"?

**Review by kayayarai — COMMENTED** [review]
I gave this an edit and (weirdly) added a screen cap that I thought was needed. Feel free to adjust. Please note that I changed the URL for the "Flat 1% Sample" section to fit style guidelines, so now it's `/tracing/trace_queries/#the-data-that-trace-queries-are-based-on`

File: content/en/tracing/trace_queries/_index.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+Query for spans from a specific environment, service, or endpoint using the [Span query syntax][1]. Use autocomplete suggestions to view facets and recent queries.
+
+Click **Add another span query** to add a span query and use it in the trace query statement.
+
+### Trace query operators
+
+Combine multiple span queries, labeled `a`, `b`, `c`, and so on, into a trace query in the **Traces where spans** field, using operators between the letters that represent each span query:
+
+{{< img src="/tracing/trace_queries/joined_span_queries.png" alt="Span queries combined into a trace query" style="width:70%;" >}}
```
Comment (dussault-antoine) [inline content/en/tracing/trace_queries/_index.md:47]:
```suggestion
{{< img src="/tracing/trace_queries/joined_span_queries.png" alt="Span queries combined into a trace query" style="width:50%;" >}}
```

File: content/en/tracing/trace_queries/_index.md (hunk 1 above)
Comment (dussault-antoine) [reply content/en/tracing/trace_queries/_index.md:68]:
```suggestion
To filter out traces that do not contain a dependency on a particular service, click on the service's node on the map.
```

**Pull request merged**